tests/Test/AbstractFixerTestCase.php

```diff
@@ -54,36 +46,22 @@ protected function setUp()
         $this->fixer = $this->createFixer();
     }
 
-    /**
-     * @return FixerInterface
-     */
-    protected function createFixer()
+    protected function tearDown()
     {
-        $fixerClassName = $this->getFixerClassName();
-
-        return new $fixerClassName();
-    }
+        parent::tearDown();
 
-    /**
-     * Create fixer factory with all needed fixers registered.
-     *
-     * @return FixerFactory
-     */
-    protected function createFixerFactory()
-    {
-        return FixerFactory::create()->registerBuiltInFixers();
+        $this->linter = null;
```

Comment:
sometimes we follow `unset($this->foo)`, sometimes `$this->foo = null`.
any reason why to do it 2 different ways ?

Reply:
I'm just used to clear properties by assigning `null` to them.

I was about to say that I'm going to use `unset()` everywhere but I checked something: https://3v4l.org/aNCUO. As you can see, `unset()` completely destroys properties just like it completely destroys regular variables, so I suggest using `$this->foo = null` everywhere instead.

Reply:
Sounds like new Fixer....

Reply:
` I suggest using $this->foo = null everywhere instead.` :+1:

Reply:
No Julien, your changes are great. Just saying that new rule would be nice, so we would never do unset on property again ;)

Reply:
I think an exception is OK here, we've enough PR's to process as it :)

Reply:
`unset()` calls on properties removed (actually only one found on 2.2, didn't check higher branches).

Reply:
don't do it, we will do higher branches with new fixer

Reply:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3695